code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -4,18 +4,26 @@
 
 /obj/item/weapon/reagent_containers/hypospray
 	name = "hypospray"
-	desc = "The DeForest Medical Corporation hypospray is a sterile, air-needle autoinjector for rapid administration of drugs to patients."
+	desc = "The DeForest Medical Corporation hypospray is a sterile, air-needle autoinjector for administration of drugs to patients."
 	icon = 'icons/obj/syringe.dmi'
 	item_state = "hypo"
 	icon_state = "hypo"
 	amount_per_transfer_from_this = 5
 	unacidable = 1
-	volume = 30
+	volume = 15
 	possible_transfer_amounts = null
 	flags = OPENCONTAINER
 	slot_flags = SLOT_BELT
 	center_of_mass = null
 	var/armorcheck = 1
+	var/time = 3 SECONDS

```

Comment:
I'm surprised this shit compiled. It's a statement, though I guess a constant.

Reply:
Macros are wonderful.